core/app/models/spree/store.rb

```diff
@@ -94,7 +106,7 @@ class Store < Spree::Base
     before_destroy :validate_not_last, unless: :skip_validate_not_last
     before_destroy :pass_default_flag_to_other_store
 
-    scope :by_url, ->(url) { where('url like ?', "%#{url}%") }
+    scope :by_url, ->(url_param) { i18n { url.matches("%#{url_param}%") } }
```

Comment:
Do we need this one, since you're no longer translating `url`?

Reply:
Aha, good catch. I'll remove.